war/gulpfile.js

```diff
@@ -1,6 +1,3 @@
-var gulp = require('gulp');
-var jshint = require('gulp-jshint');
```

Comment:
It is not needed, the key part here is to use the configuration provided by `js-builder`.